llvm/lib/Transforms/Instrumentation/SanitizerCoverage.cpp

```diff
@@ -816,12 +821,30 @@ Value *ModuleSanitizerCoverage::CreateFunctionLocalGateCmp(IRBuilder<> &IRB) {
   return Cmp;
 }
 
+Instruction *ModuleSanitizerCoverage::CreateGateBranch(Function &F,
```

Comment:
It would be nice to have a separate NFC patch for extracting `CreateGateBranch` method.